playbooks/provision.yml

```diff
@@ -29,11 +29,5 @@
     - role: webserver
       when: not development_environment
     - role: cacheserver
-    - role: DavidWittman.redis
-      vars:
-        redis_version: 4.0.9
```

Comment:
we end up with the same redis version. That's the one supported in Ubuntu bionic.